Error in callback <bound method _WandbInit._pause_backend of <wandb.sdk.wandb_init._WandbInit object at 0x7f2108100bb0>> (for post_run_cell), with arguments args (<ExecutionResult object at 7f2108100b80, execution_count=2 error_before_exec=None error_in_exec=None info=<ExecutionInfo object at 7f2108100970, raw_cell="import wandb
import pandas as pd
# Note that we us.." store_history=True silent=False shell_futures=True cell_id=d15c1a6c-7116-489d-ac98-3568fdee2537> result=None>,),kwargs {}:
Error in callback <bound method _WandbInit._resume_backend of <wandb.sdk.wandb_init._WandbInit object at 0x7f2108100bb0>> (for pre_run_cell), with arguments args (<ExecutionInfo object at 7f20d1a1b2e0, raw_cell="import wandb
import pandas as pd
# Note that we us.." store_history=True silent=False shell_futures=True cell_id=d15c1a6c-7116-489d-ac98-3568fdee2537>,),kwargs {}: